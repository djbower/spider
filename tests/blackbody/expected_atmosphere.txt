description: dS/dr (basic nodes)
scaling: 0.0004697889028520918
val:                0
val: -1.786005563846272
val: -1.728783468331907
val: -1.674343284801752
val: -1.62243698261329
val: -1.572864960202258
val: -1.525401946976837
val: -1.479879591170297
val: -1.436134207680482
val: -1.394013272765535
val: -1.353402816112942
val: -1.314164392434023
val: -1.276203289941471
val: -1.239416756927923
val: -1.20373886770353
val: -1.169072838654213
val: -1.135348828610804
val: -1.102479546415185
val: -1.07041264720718
val: -1.0390870358428
val: -1.008449859428304
val: -0.9784478353201777
val: -0.9490191548101337
val: -0.9193528434503146
val: -0.8635718623754345
val: -0.6334989917008166
val: -0.4506339050430966
val: -0.3831508317340959
val: -0.3636378203738485
val: -0.3578189576514891
val: -0.3555332445849481
val: -0.3541067733121743
val: -0.3526286061224075
val: -0.350817551163063
val: -0.3488001736781028
val: -0.3465741797556675
val: -0.3439661373929285
val: -0.3411811314998237
val: -0.3382776178005166
val: -0.3350543558588759
val: -0.3316847891251634
val: -0.3282353152952348
val: -0.3245614727084096
val: -0.3208620856731949
val: -0.3170995232927328
val: -0.3132166830098062
val: -0.3093901865788484
val: -0.3055066725002303
val: -0.3015997818727725
val: -0.2978143842771978
val: -0.293957331778179
val: -0.2901971263803669
val: -0.2865049294936121
val: -0.2828290122554432
val: -0.2793014599188053
val: -0.2757848279323692
val: -0.2723859705359374
val: -0.2690760558180119
val: -0.2658240755800794
val: -0.262714346049286
val: -0.2596382260502577
val: -0.2566984753654689
val: -0.2538155927981103
val: -0.2510303326404078
val: -0.2483262606777042
val: -0.2456957733434586
val: -0.2431615652884501
val: -0.2406722319067934
val: -0.2382774951880335
val: -0.2359175403132
val: -0.2336490915318511
val: -0.231405425852749
val: -0.2292304512135838
val: -0.227078194434133
val: -0.2249837433998342
val: -0.2229056451449929
val: -0.2208632823793368
val: -0.2188267686350801
val: -0.2168156537255335
val: -0.2148036305089933
val: -0.2127963060540006
val: -0.2107702840570087
val: -0.2087324863853783
val: -0.2066705727712703
val: -0.2045765265503023
val: -0.2024335932479961
val: -0.2002397695594202
val: -0.1979815961098366
val: -0.1956495920297494
val: -0.193217914460812
val: -0.190688684818919
val: -0.1880333248076705
val: -0.1852409560493623
val: -0.1822818494113234
val: -0.1791286479958598
val: -0.1757662007984355
val: -0.1721433114701
val: -0.1682301479967668
val: -0.1639639900471231
val:                0
description: S at surface
scaling: 0.0004697889028520918
val: 0.3691303275682391
description: Magma Ocean CO2 content
scaling: 0.0004697889028520918
val: 0.3370097439286688
description: Magma Ocean H2O content
scaling: 0.0004697889028520918
val: 60.83994792869485
