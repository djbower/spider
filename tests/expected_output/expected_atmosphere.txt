description: dS/dr (basic nodes)
scaling: 0.0004697889028520918
val:                0
val: -1.918114344069875
val: -1.853567565610742
val: -1.792545010239838
val: -1.734673472538441
val: -1.679689437667278
val: -1.627303576158893
val: -1.577234559468251
val: -1.529334393628266
val: -1.483431732344635
val: -1.439381578456515
val: -1.397005582804627
val: -1.356197754202505
val: -1.316812277561286
val: -1.278747510428051
val: -1.241902703708863
val: -1.206188277528052
val: -1.171516938939901
val: -1.137814309892214
val: -1.105006842370772
val: -1.073031452586691
val: -1.041823144266093
val: -1.011329208215618
val: -0.9814881942810313
val: -0.9522431596093945
val: -0.9230195097533581
val: -0.8778026366668634
val: -0.6845519972831692
val: -0.4840983326918574
val: -0.4028892666868805
val: -0.3778014295349659
val: -0.3703544192418773
val: -0.3675531435033552
val: -0.3657267939584917
val: -0.3640479596145891
val: -0.3622379727312489
val: -0.3600390959080348
val: -0.3576277830345085
val: -0.3550543528039367
val: -0.3521239920404656
val: -0.3490466630023825
val: -0.3458569632378646
val: -0.3424135959721501
val: -0.3389138593196629
val: -0.3353205093644784
val: -0.3315802307342268
val: -0.3278720612300793
val: -0.3240824183493702
val: -0.3202448523810423
val: -0.3165090669108004
val: -0.3126832307332322
val: -0.3089339513790063
val: -0.3052354339733785
val: -0.3015371795640161
val: -0.2979727371488508
val: -0.2944054875980459
val: -0.2909438412048154
val: -0.2875608125487236
val: -0.2842257789940922
val: -0.2810241656519026
val: -0.2778489237599382
val: -0.2748028353222898
val: -0.2718083386019708
val: -0.2689067663693326
val: -0.2660831717154133
val: -0.2633292973196165
val: -0.2606680632015745
val: -0.2580521812398079
val: -0.2555308868436261
val: -0.2530457439060599
val: -0.2506506279466545
val: -0.2482829157873075
val: -0.2459863082964295
val: -0.2437163364977573
val: -0.2415073199742749
val: -0.2393191682613863
val: -0.237172625047231
val: -0.2350389360840239
val: -0.2329367674786841
val: -0.230840240598795
val: -0.228757653707763
val: -0.226667548762573
val: -0.2245759658099584
val: -0.2224715910012413
val: -0.2203472278418008
val: -0.2181884903748011
val: -0.2159949127722521
val: -0.213754377342088
val: -0.2114592268653402
val: -0.2090861430484048
val: -0.2066384277315684
val: -0.2040898557817507
val: -0.2014349540692476
val: -0.1986483101789109
val: -0.1957060413816831
val: -0.1925986181704657
val: -0.1892794642130992
val: -0.1857313650400914
val: -0.1819071234355541
val:                0
description: S at surface
scaling: 0.0004697889028520918
val: 0.351193564144183
description: Magma ocean volatile content
scaling: 0.0004697889028520918
val: 0.2076888998639458
val: 59.91299575467099
